runtime/frequency/src/lib.rs

```diff
@@ -549,6 +565,15 @@ impl pallet_capacity::Config for Runtime {
 	type EpochNumber = u32;
 	type CapacityPerToken = CapacityPerToken;
 	type RuntimeFreezeReason = RuntimeFreezeReason;
+	type EraLength = CapacityRewardEraLength;
+	type ProviderBoostHistoryLimit = ConstU32<30u32>;
```

Comment:
Shouldn't this refer to the `parameter_types!` value?
```suggestion
	type ProviderBoostHistoryLimit = ProviderBoostHistoryLimit;
```

Reply:
Yes.